libs/community/langchain_community/llms/anthropic.py

```diff
@@ -92,11 +95,13 @@ def validate_environment(cls, values: Dict) -> Dict:
                 base_url=values["anthropic_api_url"],
                 api_key=values["anthropic_api_key"].get_secret_value(),
                 timeout=values["default_request_timeout"],
+                max_retries=values["max_retries"],
```

Comment:
believe anthropic.Anthropic() doesn't accept None for max_retries, should we give it same default as they do (2)?

Reply:
Agreed, looks None isn't valid here. Updated to match their implementation.

Reply:
What do you think @baskaryan ?